output/eu_crude/analysis.py

```diff
@@ -74,8 +78,148 @@ def get_waste_id(resource_array):
     return set(wasteid)
 
 
+def get_colors():
+    """ Gets colors from a matplotlib dictionary
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    Colores: array
+        array of color hex codes
+    """
+
+    colores = []
+    for hex in matplotlib.colors.cnames.items():
```

Comment:
also, you would get much nicer colors if you didn't just sort through the matplotlib available colors at random and instead used a colormap like viridis. Also, FYI, you can use rgb2hex(). 

```
import matplotlib 
from matplotlib import cm
n_countries = 20
cmap = cm.get_cmap('viridis', n_countries)
for i in range(cmap.N):
    rgb = cmap(i)[:3] # will return rgba, we take only first 3 so we get rgb
    print(matplotlib.colors.rgb2hex(rgb))
```